app/views/vacancies/index.html.haml

```diff
@@ -5,23 +5,23 @@
   = content_for :page_title_prefix, t('jobs.heading')
 = render partial: 'head_links'
 
-%h1.govuk-heading-xl
-  = t('jobs.heading')
+%h1.govuk-heading-xl#job-count
+  -# = t('jobs.heading')
+  = location_category_content?(@filters) ? @vacancies.total_count_message_with_location(format_location_name(@filters.location)) : @vacancies.total_count_message
 .govuk-grid-row
   .govuk-grid-column-one-third
     = render 'filters'
   .govuk-grid-column-two-thirds
     .govuk-grid-column-full.vacancies-count
-      %p.govuk-heading-l.mb0.inline-block#vacancy-results= location_category_content?(@filters) ? @vacancies.total_count_message_with_location(format_location_name(@filters.location)) : @vacancies.total_count_message
       - if @vacancies.user_search?
         %span.clear-search.nobreak= link_to t('jobs.filters.clear_filters'), root_path, class: 'govuk-link'
         = render(partial: 'subscribe_link', locals: { search_criteria: @filters.only_active_to_hash }) if EmailAlertsFeature.enabled? && !ReadOnlyFeature.enabled?
     - if @vacancies.any?
       = render partial: 'sorting_options', locals: { search_criteria: @filters.only_active_to_hash, sort: @sort }
-
-      %ul.vacancies
-        - @vacancies.each do |vacancy|
-          = render partial: 'vacancy', locals: { vacancy: vacancy }
+      #vacancies-hits
```

Comment:
Can you just have `%ul.vacancies#vacancies-hits` ?

Reply:
possibly, its on todo list in this PR to go through CSS selectors/markup in general as im not happy with it in places, took a few shortcuts just to get it to 'work'